examples/spec/integration/upsert_search_attributes_spec.rb

```diff
@@ -31,7 +31,8 @@
       workflow_id: workflow_id,
       run_id: run_id,
     )
-    expect(added_attributes).to eq(expected_added_attributes)
+    expect(added_attributes[:from_code]).to eq(expected_added_attributes)
```

Comment:
nit: Appreciate keeping the existing test coverage in place, but it's probably not necessary--just drop the `:from_code` part and change the example?

Reply:
Ended up refactoring this a bit to make it more straightforward and useful in verifying correctness